### DISCUSSION TITLE
Are there FAILs being ignored?

### DISCUSSION THREAD

**Discussion by felipesanches** [fonttools/fontbakery#3291] [General]
A question to anyone who's been using fontbakery on a daily basis:

Please list here any FB check that results in a FAIL for your font projects and that you ignore for some reason. I'd like to review them and either downgrade them to WARN or discuss whether we could change their implementation.

The goal here is to make sure we never ignore FAILs, ever! So we need to address any instances of this practice.

**Comment by m4rc1e** [comment]
- Ligature carets (has been fixed)
- Named Stylistic Sets for legacy families
- Avar if axes are linear
- Prep and GASP from 3rd party families since they most likely have their own settings which render better

**Reply by m4rc1e** [reply]
For 3rd party families, I ignore superfluous name entries such as the copyright strings. As long as they are using the RIBBI schema, I'm happy.

**Comment by RosaWagner** [comment]
- camel case naming
- localized names when not latin script
- All the copyright string fails are being ignored (I do think that we shouldn't, but when the font arrives like this on google/fonts repo, it is not a good reason to send the font back through the entire process again, cause in general it is just a question of the licence's URL which is `http` instead of `https`)

**Comment by chrissimpkins** [comment]
- com_google_fonts_check_dsig (the MS Office version isn't in our use case targets)

**Comment by chrissimpkins** [comment]
> The goal here is to make sure we never ignore FAILs, ever! So we need to address any instances of this practice.

Can you explain what this means?  Do you intend to prevent projects from filtering out FAIL level checks that are not applicable?

**Comment by RosaWagner** [comment]
I would like to also propose a WARN:

This check `⚠ WARN: Is there kerning info for non-ligated sequences?` is systematically ignored for multiple reasons:

1. some ligature are not of letters (example` -->` to create an arrow)
2. in the case of ligature `tt` for example, it can be good to keep the option of not having the letters touching, by deactivating the ligature.
4. ligatures are often ornemental which doesn't necessitate a kerned sequenced from the base pair

Since this check apply only for edge cases and not for common usage, it could be removed entirely, or only for a specific profile of a specific client.

**Reply by vv-monsalve** [reply]
Given the origin of the check is to avoid [this issue](https://github.com/impallari/Raleway/issues/14) I think the Warn is useful and yes, it would rely on a case by case consideration as many warns.

However, what is not functional is to [still receive the warn](https://github.com/googlefonts/fontbakery/issues/2628) even in cases when the kerning info was needed and has been added for all the cases in the font. I think then is when it gets systematically ignored.

**Comment by RosaWagner** [comment]
There is this other WARN that is always ignored:

⚠ WARN: METADATA.pb: Fontfamily is listed on Google Fonts API?
com.google.fonts/check/metadata/listed_on_gfonts

⚠ WARN Family not found via Google Fonts API. [code: not-found]
